Assets/DeathNokonoko.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DeathNokonoko : DeathBase
+{
+
+    // Update is called once per frame
+    protected new void Update()
+    {
+        base.Update();
+    }
+
+    public override void Die()
+    {
+        if (_isDead == false)
+        {
+            // 演出
+            var rb = this.GetComponent<Rigidbody2D>();
+
+            if (rb != null) rb.velocity += Vector2.up * 3f;
+            this.transform.rotation *= Quaternion.Euler(0f, 0f, 180f);
```

Comment:
```suggestion
            // 演出
            var rb = this.GetComponent<Rigidbody2D>();

            if (rb != null) rb.velocity += Vector2.up * 3f;
            this.transform.rotation *= Quaternion.Euler(0f, 0f, 180f);
```
コメントにて「演出」と書かれていますが、
どのような演出を行っているかが想像がつかないと、
分かりにくくなってしまう可能性があります。
明日は別の自分がいると思って分かりやすく書いてもらえるとありがたいです。